packages/office-ui-fabric-react/src/components/Dialog/Dialog.styles.ts

```diff
@@ -31,3 +33,18 @@ export const getStyles = (props: IDialogStyleProps): IDialogStyles => {
     ]
   };
 };
+
+export const getMergedDialogContentStyles = memoizeFunction(
+  (dialogContentProps?: IDialogContentProps, dialogDraggableClassName?: string): Partial<IDialogContentStyles> => {
```

Comment:
good point about if `dialogContentProps.styles` function. I believe that would cause issues with memoize

---
In reply to: [267969121](https://github.com/OfficeDev/office-ui-fabric-react/pull/8426#discussion_r267969121) [](ancestors = 267969121)